scripts/update-argo-workflows-crds.sh

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env bash
+#
+# Update Argo Workflows CRDs from upstream
+#
+# Usage: ./scripts/update-argo-workflows-crds.sh <version>
+# Example: ./scripts/update-argo-workflows-crds.sh v3.7.4
+#
+
+set -euo pipefail
+
+VERSION="${1:-}"
+
+if [[ -z "$VERSION" ]]; then
+    echo "Usage: $0 <version>"
+    echo "Example: $0 v3.7.4"
+    exit 1
+fi
+
+# Ensure version starts with 'v'
+if [[ ! "$VERSION" =~ ^v ]]; then
+    VERSION="v${VERSION}"
+fi
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+REPO_ROOT="$(cd "$SCRIPT_DIR/.." && pwd)"
+CRD_DIR="$REPO_ROOT/charts/argo-workflows/files/crds"
+
+UPSTREAM_BASE_URL="https://raw.githubusercontent.com/argoproj/argo-workflows/${VERSION}/manifests/base/crds"
+
+# List of CRD files to download
+CRD_FILES=(
```

Comment:
❓ Could we pull this list from https://github.com/argoproj/argo-helm/tree/main/charts/argo-workflows/files/crds/full? I only ask in case a new one is added, we'll have to remember to update this list.